spec/target_spec.rb

```diff
@@ -17,6 +17,30 @@ module Xcake
         end
       end
 
+      context "for tvos" do
+
+        before :each do
+          @target = Target.new
+          @target.platform = :tvos
+        end
+
+        it "should set default system frameworks" do
+          expect(@target.system_frameworks).to eq(['Foundation', 'UIKit'])
+        end
+      end
+
+      context "for watchos" do
+
+        before :each do
+          @target = Target.new
+          @target.platform = :watchos
+        end
+
+        it "should set default system frameworks" do
+          expect(@target.system_frameworks).to eq(['Foundation', 'UIKit', 'WatchKit'])
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%w` or `%W` for array of words.